pkg/operator/apiserver/controller/workload/workload.go

```diff
@@ -273,6 +273,34 @@ func (c *Controller) updateOperatorStatus(workload *appsv1.Deployment, operatorC
 		Status: operatorv1.ConditionFalse,
 	}
 
+	if !preconditionsReady {
+		message := "the operator haven't specified what preconditions are missing"
```

Comment:
nit: either "hasn't specified which" but I think "did not specify which" is more correct